sgx.mk

```diff
@@ -221,15 +222,20 @@ sgx/util.o: src/util/util.c
 	@$(CXX) $(Enclave_Cpp_Flags) -c $< -o $@
 	@echo "CXX  <= $<"
 
-sgx/pelz_enclave.so: sgx/pelz_enclave_t.o sgx/key_table.o sgx/aes_keywrap_3394nopad.o sgx/pelz_request_handler_impl.o sgx/charbuf.o sgx/util.o
+sgx/$(Enclave_Name): sgx/pelz_enclave_t.o sgx/key_table.o sgx/aes_keywrap_3394nopad.o sgx/pelz_request_handler_impl.o sgx/charbuf.o sgx/util.o
 	@$(CXX) $^ -o $@ $(Enclave_Link_Flags)
 	@echo "LINK =>  $@"
 
-sgx/$(Signed_Enclave_Name): sgx/$(Enclave_Name)
-	@$(SGX_ENCLAVE_SIGNER) sign -key sgx/pelz_enclave_private.pem -enclave sgx/$(Enclave_Name) -out $@ -config $(Enclave_Config_File)
+sgx/$(Enclave_Signing_Key):
```

Comment:
Rather than having this target echo a message, it might be clearer to use a `Make` error so processing stops and the user doesn't have to look through a bunch of output to find these lines. Somewhere earlier in the Makefile define the error message
```
define err_no_enclave_signing_key
FAIL: No Enclave signing key found - must generate or install sgx/$(Enclave_Signing_Key)
      e.g., run 'openssl genrsa -out sgx/$(Enclave_Signing_Key) -b 3072'
endef
```
and then the target here can just be 
```
$(error $(err_no_enclave_signing_key))
```

Reply:
Yes, this is a much better way to handle this. I have incorporated these changes.